src/cron.ts

```diff
@@ -183,18 +208,25 @@ export const cron = async (startDate: Date, owner: string, repo: string, octokit
       }
 
       // if we don't find a match, we search for a closed referenda
-      const completedMatch = completedReferendas.find(
+      const completedMatch = finishedReferendas.find(
         ({ hash }) => hash === tx.method.hash.toHex() || hash === tx.method.toHex(),
       );
       if (completedMatch) {
-        logger.info(`Found completed referenda for PR #${pr}`);
+        logger.info(`Found finished referenda for PR #${pr} with state ${completedMatch.state}`);
         const command = `/rfc process ${completedMatch.executedHash}`;
-        const msg = "PR can be merged. Write the following command to trigger the bot" + `\n\n\`${command}\``;
+        const rejectedMsg = `Referenda voting has fnished with status \`${completedMatch.state}\``;
+        const finishedMsg =
+          "PR can be merged." + "\n\nWrite the following command to trigger the bot\n\n" + `\`${command}\``;
         rows.push([
           `${owner}/${repo}#${pr}`,
           `<a href="https://collectives.polkassembly.io/referenda/${completedMatch.index}>RFC ${completedMatch.index}</a>`,
         ]);
-        await octokit.rest.issues.createComment({ owner, repo, issue_number: pr, body: msg });
+        await octokit.rest.issues.createComment({
+          owner,
+          repo,
+          issue_number: pr,
+          body: completedMatch.state === "Executed" ? finishedMsg : rejectedMsg,
```

Comment:
Shouldn't it be `Confirmed`, not `Executed`?

Reply:
No, because this is the state of the referendum, not of the block.

It is using [this field](https://github.com/paritytech/rfc-action/blob/8d91dd25d0f357819d9973ca656f1490c554222c/src/subsquare.ts#L15)
```ts
  state: {
    name: "Executed" | "Rejected" | "TimedOut" | string;
  };
```